workloads/network-perf/README.md

```diff
@@ -17,35 +17,52 @@ $ ./run_<test-name>_network_test_fromgit.sh
 ## Environment variables
 
 ### ES_USER
-Default: `` 
+Default: ``    
 Username for elasticsearch instance
 
 ### ES_PASSWORD
-Default: `` 
+Default: ``     
 Password for elasticsearch instance
 
 ### ES_SERVER
-Default: `milton.aws.com`  
+Default: `milton.aws.com`    
 Elasticsearch server to index the results of the current run
 
 ### ES_PORT
-Default: ``  
+Default: ``    
 Port number for elasticsearch server
 
 ### METADATA_COLLECTION
 Default: `true`   
 Enable/Disable collection of metadata
 
 ### COMPARE
-Default: `false`   
+Default: `false`        
```

Comment:
Something happened here... you added random spaces?

Reply:
The next line continues after `false` if we don't add enough spaces